### ISSUE TITLE
Simple demo doable (just plain JS)?

### ISSUE THREAD

**Issue by k00ni** [AKSW/cube-validator#5]
@GeorgesAlkhouri: For [such cases](https://github.com/AKSW/cubevizjs/issues/24#issuecomment-284788679) the cube-validator would be handy. 

I thought about me building a simple demo-page, but without the Docker setup. Can you tell me, if there is anything to care about before? Can i use the generated main.js, do some fancy JS and put that on a page?

**Comment by GeorgesAlkhouri** [comment]
Yes, this is now possible. See https://github.com/AKSW/cube-validator#in-browser.

**Comment by k00ni** [comment]
What is missing:
* [ ] integrate [Fuseki](https://hub.docker.com/r/stain/jena-fuseki/) as triple store
